modules/lib/lib-http-client/src/test/java/com/enonic/xp/lib/http/HttpRequestHandlerTest.java

```diff
@@ -100,6 +100,20 @@ public void testSimplePostRequest()
     }
 
     @Test
+    public void testSimpleHeadRequest()
+        throws Exception
+    {
+        server.enqueue( addResponse( "GET request" ) );
```

Comment:
No. Because the content length returned by a HEAD request should be the same as a GET request.

Check mate.